examples/existing-instance/main.tf

```diff
@@ -0,0 +1,21 @@
+########################################################################################################################
+# Resource Group
+########################################################################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.6"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+########################################################################################################################
+# Watsonx Data
+########################################################################################################################
+
+module "existing_watsonx_data_instance" {
+  source                             = "../../"
+  existing_watsonx_data_instance_crn = var.existing_watsonx_data_instance_crn
+  resource_group_id                  = module.resource_group.resource_group_id
```

Comment:
why would you pass `resource_group_id` here? Its only used when creating a new instance